codecs/oxipng/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+rm -rf pkg,{-parallel}
+wasm-pack build
+RUSTFLAGS='-C target-feature=+atomics,+bulk-memory' wasm-pack build -t web -d pkg-parallel -- -Z build-std=panic_abort,std --features=parallel
+sed -i "s|input = import.meta.url.replace(/\\\.js$/, '_bg.wasm');||" pkg{,-parallel}/squoosh_oxipng.js
```

Comment:
What is happening here?

Reply:
I'm removing an autogenerated line "input = import.meta.url.replace(/\\\.js$/, '_bg.wasm');" to work around an issue in Webpack (it doesn't like / recognise `import.meta` syntax). There are other ways to do this, but this one requires the least effort, and we hopefully won't have the same problem with other toolchains anyway.

Reply:
Okay, let’s ignore this you are correctly pointing out that we are moving to Rollup anyway.